src/main/java/Parser.java

```diff
@@ -0,0 +1,35 @@
+import java.io.IOException;
+
+
+public class Parser {
```

Comment:
String manipulation and any information extraction from the user input can be done in Parser instead of Ui.